documentation/API proposals/APIproposal_QualityByDesign_CableLabs.md

```diff
@@ -0,0 +1,16 @@
+|                                           | **API Scope**<br>This API would be part of the Home Device API Family. The QbD API allows a 3rd party application developer to create a session with a network operator. The application can then share KPIs such as:<br>- latency<br>- jitter<br>- packet loss<br>- bitrate<br>The network operator can monitors these KPIs and determine a QbD score. If the score ever drop below certain thresholds, an event can be triggered for rapid telemetry collection. The network operator can then determine root cause and suggest corrective actions either to the 3rd party app or to the network. |
+|                                           | **Use Case Example 1 - Wi-Fi Congestion**<br>Wi-Fi congestion at a `Service Site` is affecting video conferencing performance, reflecting lower QbD scores and poor video quality                                                                                                                                                                                                                                                                                                                                                                                                                   |
+|                                           | **Use Case Example 2 - Network Impairments**<br>Issues in network operator's network can lead to degradation in latency, jitter, packet loss, and bitrate, reflecting lower QbD scores and poor video quality.                                                                                                                                                                                                                                                                                                                                                                                      |
+|                                           | **Sample Recommendations**<br>- speed boost<br>- traffic prioritization<br>- WMM enablement<br>- application configuration updates<br>- request service improvements                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
I don't think that speed boost is general used in CAMARA or 3GPP.  Maybe `- Create a Quality on Demand session` and that wold cover both speed boost and traffic prioritization examples in terms of CAMARA APIs.

It's ok to use these in the slides where you explain them further, but keeping this to terms relevant to CAMARA can help convey the overall goal more easily.  Maybe add (HomeDeviceQoD Session) to WMM enablement.  

@shafayetkhan does that fit as well?

Reply:
ok, made updates, will be pushing PR update soon

Reply:
Yes, I like that @RandyLevensalor !